src/api/test/protectedApiClient.test.ts

```diff
@@ -22,20 +22,4 @@ describe('Protected API Client Tests', () => {
       expect(result).toEqual(response);
     });
   });
-
-  describe('deleteUser', () => {
```

Comment:
no, we like tests

Reply:
Not tests for functions that no longer exist!